packages/extension-base/src/koni/background/handlers/State.ts

```diff
@@ -1177,6 +1182,345 @@ export default class KoniState {
+
+    const payload: BitcoinSignPsbtPayload = {
+      psbt: psbtGenerate,
+      broadcast: !!broadcast,
+      network,
+      signAtIndex: isArray(signAtIndex) && signAtIndex.length === 0 ? undefined : signAtIndex,
+      account: account.address,
+      allowedSighash,
+      txInput: psbtTxInputs,
+      txOutput: psbtTxOutputs
+    };
+    const hashPayload = '';
+    const canSign = !account.isExternal;
+
+    const signPayload: BitcoinSignPsbtRequest = {
+      account,
+      payload,
+      hashPayload,
+      canSign
+    };
+
+    return this.requestService.addConfirmationBitcoin(id, url, 'bitcoinSignPsbtRequest', signPayload, {
+      requiredPassword: false
+    })
+      .then(({ isApproved, payload }) => {
+        if (isApproved) {
+          if (payload) {
+            return payload;
+          } else {
+            throw new BitcoinProviderError(BitcoinProviderErrorType.INVALID_PARAMS, t('Not found signature'));
+          }
+        } else {
+          throw new BitcoinProviderError(BitcoinProviderErrorType.USER_REJECTED_REQUEST);
+        }
+      });
+  }
+
+  public async bitcoinSendTransaction (id: string, url: string, networkKey: string, allowedAccounts: string[], transactionParams: BitcoinSendTransactionParams): Promise<string | undefined> {
+    const bitcoinApi = this.getBitcoinApi(networkKey);
+    const apiStrategy = bitcoinApi.api;
+
+    const autoFormatNumber = (val?: string | number): string | undefined => {
+      if (typeof val === 'string' && val.startsWith('0x')) {
+        return new BigN(val.replace('0x', ''), 16).toString();
+      } else if (typeof val === 'number') {
+        return val.toString();
+      }
+
+      return val;
+    };
+
+    if (transactionParams.recipients.length !== 1) {
+      throw new BitcoinProviderError(BitcoinProviderErrorType.INVALID_PARAMS, t('Receiving address must be a single account'));
+    }
+
+    if (transactionParams.account === transactionParams.recipients[0].address) {
+      throw new BitcoinProviderError(BitcoinProviderErrorType.INVALID_PARAMS, t('Receiving address must be different from sending address'));
+    }
+
+    const transaction: BitcoinTransactionConfig = {
+      from: transactionParams.account,
+      to: transactionParams.recipients[0].address,
+      value: autoFormatNumber(transactionParams.recipients[0].amount),
+      networkKey: transactionParams.network
+    };
+
+    // Address is validated in before step
+    const fromAddress = allowedAccounts.find((account) => (account.toLowerCase() === (transaction.from as string).toLowerCase()));
+
+    if (!fromAddress) {
+      throw new EvmProviderError(EvmProviderErrorType.INVALID_PARAMS, t('You have rescinded allowance for this account in wallet'));
+    }
+
+    const pair = keyring.getPair(fromAddress);
+
+    if (!pair) {
+      throw new EvmProviderError(EvmProviderErrorType.INVALID_PARAMS, t('Unable to find account'));
+    }
+
+    if (networkKey === 'mainnet') {
```

Comment:
The `networkKey` has been converted to `chainSlug`, so this condition is incorrect